### ISSUE TITLE
bring coverage to 100%

### ISSUE THREAD

**Issue by jugmac00** [jugmac00/flask-reuploaded#73]
only a couple of lines missing

```
Name                                           Stmts   Miss  Cover   Missing
----------------------------------------------------------------------------
src/flask_uploads/__init__.py                     23      0   100%
src/flask_uploads/backwards_compatibility.py       2      0   100%
src/flask_uploads/exceptions.py                    1      0   100%
src/flask_uploads/extensions.py                   31      0   100%
src/flask_uploads/flask_uploads.py               152      8    95%   64, 84, 166, 197, 220-221, 314, 379
src/flask_uploads/test_helper.py                  11      0   100%
tests/__init__.py                                  0      0   100%
tests/test_autoserve.py                           90      0   100%
tests/test_flask_reuploaded.py                   248      0   100%
----------------------------------------------------------------------------
TOTAL     
```

**Comment by jugmac00** [comment]
```
Name                                           Stmts   Miss  Cover   Missing
----------------------------------------------------------------------------
src/flask_uploads/__init__.py                     23      0   100%
src/flask_uploads/backwards_compatibility.py       2      0   100%
src/flask_uploads/exceptions.py                    1      0   100%
src/flask_uploads/extensions.py                   31      0   100%
src/flask_uploads/flask_uploads.py               152      6    96%   166, 197, 220-221, 314, 379
src/flask_uploads/test_helper.py                  11      0   100%
tests/__init__.py                                  0      0   100%
tests/test_autoserve.py                           90      0   100%
tests/test_flask_reuploaded.py                   254      0   100%
----------------------------------------------------------------------------
TOTAL                                            564      6    99%
```

**Comment by jugmac00** [comment]
```
Name                                           Stmts   Miss  Cover   Missing
----------------------------------------------------------------------------
src/flask_uploads/__init__.py                     23      0   100%
src/flask_uploads/backwards_compatibility.py       2      0   100%
src/flask_uploads/exceptions.py                    1      0   100%
src/flask_uploads/extensions.py                   31      0   100%
src/flask_uploads/flask_uploads.py               152      5    97%   197, 220-221, 314, 379
src/flask_uploads/test_helper.py                  11      0   100%
tests/__init__.py                                  0      0   100%
tests/test_autoserve.py                           90      0   100%
tests/test_flask_reuploaded.py                   257      0   100%
----------------------------------------------------------------------------
TOTAL    
```

**Comment by jugmac00** [comment]
```
Name                                           Stmts   Miss  Cover   Missing
----------------------------------------------------------------------------
src/flask_uploads/__init__.py                     23      0   100%
src/flask_uploads/backwards_compatibility.py       2      0   100%
src/flask_uploads/exceptions.py                    1      0   100%
src/flask_uploads/extensions.py                   31      0   100%
src/flask_uploads/flask_uploads.py               152      4    97%   220-221, 314, 379
src/flask_uploads/test_helper.py                  11      0   100%
tests/__init__.py                                  0      0   100%
tests/test_autoserve.py                           90      0   100%
tests/test_flask_reuploaded.py                   260      0   100%
----------------------------------------------------------------------------
TOTAL                                            570      4    99%
```

**Comment by jugmac00** [comment]
```
Name                                           Stmts   Miss  Cover   Missing
----------------------------------------------------------------------------
src/flask_uploads/__init__.py                     23      0   100%
src/flask_uploads/backwards_compatibility.py       2      0   100%
src/flask_uploads/exceptions.py                    1      0   100%
src/flask_uploads/extensions.py                   31      0   100%
src/flask_uploads/flask_uploads.py               152      3    98%   220-221, 379
src/flask_uploads/test_helper.py                  11      0   100%
tests/__init__.py                                  0      0   100%
tests/test_autoserve.py                           90      0   100%
tests/test_flask_reuploaded.py                   265      0   100%
----------------------------------------------------------------------------
TOTAL                                            575      3    99%
```

**Comment by jugmac00** [comment]
fixed by #74
